src/routes/Messages/Messages.js

```diff
@@ -19,6 +19,20 @@ import './Messages.scss'
 
 export const NEW_THREAD_ID = 'new'
 
+const promptLookup = {
+  research: '[Research projects] ',
```

Comment:
lemme just the copy by clare one more time otherwise looks good to go!

Reply:
Sure thing